src/python/pants/backend/jvm/tasks/jvm_compile/rsc/rsc_compile.py

```diff
@@ -428,7 +428,7 @@ def make_cache_doublecheck_job(dep_keys):
       # depends on. It depends on completion of the same dependencies as the rsc job in order to run
       # as late as possible, while still running before rsc or zinc.
       return Job(cache_doublecheck_key,
-                 functools.partial(self._default_double_check_cache_for_vts, ivts),
```

Comment:
`_default_double_check_cache_for_vts` is also used in rsc compile? Do you think they can both adopt the new `_double_check_cache_for_vts` instead?

Reply:
Yes, absolutely! Will do!

Reply:
Done!

Reply:
Hey, so I've moved this back to `rsc_compile.py` because the `zinc_compile_context` that it needs is only available in the `RscCompile#create_compile_jobs()` method, not in the regular `jvm_compile.py`.